projects/ngx-charts-on-fhir/src/lib/fhir-chart-summary/fhir-chart-summary/fhir-chart-summary.component.spec.ts

```diff
@@ -67,20 +67,20 @@ describe('FhirChartSummaryComponent', () => {
     const layer: ManagedDataLayer = { id: '1', name: 'layer', datasets: [], scale: { id: 'test' } };
     layerManager.selectedLayers$.next([layer, layer, layer]);
     fixture.detectChanges();
-    const cards = fixture.debugElement.queryAll(By.directive(MockFhirChartSummaryCardComponent));
+    const cards = fixture.debugElement.queryAll(By.css('.summary-card'));
```

Comment:
Why did you change this? we still want to make the sure the cards are rendered and not just empty divs, right?
Also, please add a test for `it('should not render a card for disabled layers')`